rero_ils/theme/assets/js/reroils/login.js

```diff
@@ -0,0 +1,59 @@
+/*
+
+RERO ILS
+Copyright (C) 2019 RERO
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, version 3 of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+import $ from 'jquery';
+
+$("#login-user").submit(function (e) {
+
+  e.preventDefault(); // avoid to execute the actual submit of the form.
+
+  var form = $(this);
+  var url = form.attr('data-action');
+
+  $.ajax({
+    type: "POST",
+    url: url,
+    data: form.serialize(), // serializes the form's elements.
+    success: function (data) {
+      var next = getUrlParameter('next');
+      window.location.replace(next ? next : '/');
+    },
+    error: function (data) {
+      var response = $.parseJSON(data.responseText);
+      var alert = $("#js-alert");
+      var msg = $("#js-alert span.msg").html(response.errors[0].message);
+      alert.show();
+    }
+  });
+});
+
+var getUrlParameter = function getUrlParameter(sParam) {
+  var sPageURL = window.location.search.substring(1),
+    sURLVariables = sPageURL.split('&'),
+    sParameterName,
+    i;
```

Comment:
Mmmm.... strange construction, I would trust you

Reply:
Me too, I found this code on the web and seems work.